package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "build-all": "npm-run-posix-or-windows build-all:task",
-    "build-all:task": "(yarn build-cdt-lib & yarn build-extension & yarn build-devtools & yarn build-lr & yarn build-viewer & yarn build-smokehouse-bundle & wait) && yarn build-pack",
+    "build-all:task": "yarn build-cdt-lib && yarn build-devtools && (yarn build-extension & yarn build-lr & yarn build-viewer & yarn build-smokehouse-bundle & wait) && yarn build-pack",
```

Comment:
build-cdt-lib could be in the parallel group though, right?

Reply:
so the results of cdt-lib are actually checked into git. if the output varies at all from what is in source control then CI will fail.

I made it happen before just because if you were to run this script locally, you'd want the CDT lib changes to be picked up by the build-devtools step.